src/core/audio/AudioSdl.cpp

```diff
@@ -36,6 +37,10 @@
 namespace lmms
 {
 
+constexpr char const* const c_sectionSDL = "audiosdl";
+constexpr char const* const c_playbackDeviceSDL = "device";
+constexpr char const* const c_inputDeviceSDL = "inputdevice";
```

Comment:
I believe this naming style fits our conventions better.
```suggestion
constexpr char const* const SectionSDL = "audiosdl";
constexpr char const* const PlaybackDeviceSDL = "device";
constexpr char const* const InputDeviceSDL = "inputdevice";
```